adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -142,6 +137,7 @@ func (a *adapter) MakeBids(internalRequest *openrtb.BidRequest, externalRequest
 func getMediaTypeForImp(impID string, imps []openrtb.Imp) (openrtb_ext.BidType, error) {
 	mediaType := openrtb_ext.BidTypeBanner
 	for _, imp := range imps {
+		fmt.Println(impID, imp.ID)
```

Comment:
This looks like a leftover log line for local debugging. Can you please remove it?